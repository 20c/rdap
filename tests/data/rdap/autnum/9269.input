{
  "rdapConformance": [
    "history_version_0",
    "nro_rdap_profile_0",
    "nro_rdap_profile_asn_hierarchical_0",
    "cidr0",
    "rdap_level_0"
  ],
  "notices": [
    {
      "title": "Source",
      "description": [
        "Objects returned came from source",
        "APNIC"
      ]
    },
    {
      "title": "Terms and Conditions",
      "description": [
        "This is the APNIC WHOIS Database query service. The objects are in RDAP format."
      ],
      "links": [
        {
          "value": "https://rdap.apnic.net/autnum/9269",
          "rel": "terms-of-service",
          "href": "http://www.apnic.net/db/dbcopyright.html",
          "type": "text/html"
        }
      ]
    },
    {
      "title": "Whois Inaccuracy Reporting",
      "description": [
        "If you see inaccuracies in the results, please visit: "
      ],
      "links": [
        {
          "value": "https://rdap.apnic.net/autnum/9269",
          "rel": "inaccuracy-report",
          "href": "https://www.apnic.net/manage-ip/using-whois/abuse-and-spamming/invalid-contact-form",
          "type": "text/html"
        }
      ]
    }
  ],
  "country": "HK",
  "events": [
    {
      "eventAction": "registration",
      "eventDate": "2008-09-04T06:40:30Z"
    },
    {
      "eventAction": "last changed",
      "eventDate": "2020-07-15T13:09:38Z"
    }
  ],
  "name": "HKBN-AS-AP",
  "remarks": [
    {
      "description": [
        "Hong Kong Broadband Network Ltd."
      ],
      "title": "description"
    }
  ],
  "links": [
    {
      "value": "https://rdap.apnic.net/autnum/9269",
      "rel": "self",
      "href": "https://rdap.apnic.net/autnum/9269",
      "type": "application/rdap+json"
    },
    {
      "value": "https://rdap.apnic.net/autnum/9269",
      "rel": "related",
      "href": "https://netox.apnic.net/search/AS9269?utm_source=rdap&utm_medium=result&utm_campaign=rdap_result",
      "type": "text/html"
    }
  ],
  "status": [
    "active"
  ],
  "handle": "AS9269",
  "endAutnum": 9269,
  "startAutnum": 9269,
  "objectClassName": "autnum",
  "entities": [
    {
      "roles": [
        "registrant"
      ],
      "events": [
        {
          "eventAction": "registration",
          "eventDate": "2017-08-21T12:56:42Z"
        },
        {
          "eventAction": "last changed",
          "eventDate": "2023-09-05T02:14:48Z"
        }
      ],
      "links": [
        {
          "value": "https://rdap.apnic.net/autnum/9269",
          "rel": "self",
          "href": "https://rdap.apnic.net/entity/ORG-HKBN1-AP",
          "type": "application/rdap+json"
        }
      ],
      "vcardArray": [
        "vcard",
        [
          [
            "version",
            {},
            "text",
            "4.0"
          ],
          [
            "fn",
            {},
            "text",
            "Hong Kong Broadband Network Ltd"
          ],
          [
            "kind",
            {},
            "text",
            "org"
          ],
          [
            "adr",
            {
              "label": "15/F Trans Asia Centre\n18 Kin Hong Street"
            },
            "text",
            [
              "",
              "",
              "",
              "",
              "",
              "",
              ""
            ]
          ],
          [
            "tel",
            {
              "type": "voice"
            },
            "text",
            "+852-3999-3888"
          ],
          [
            "tel",
            {
              "type": "fax"
            },
            "text",
            "+852-8167-7020"
          ],
          [
            "email",
            {},
            "text",
            "nocsn@hkbn.com.hk"
          ]
        ]
      ],
      "objectClassName": "entity",
      "handle": "ORG-HKBN1-AP"
    },
    {
      "roles": [
        "abuse"
      ],
      "events": [
        {
          "eventAction": "registration",
          "eventDate": "2012-05-16T06:32:40Z"
        },
        {
          "eventAction": "last changed",
          "eventDate": "2023-10-27T01:46:39Z"
        }
      ],
      "remarks": [
        {
          "description": [
            "abuse@hkbn.net was validated on 2023-09-13",
            "hostmaster@hkbn.com.hk was validated on 2023-10-27"
          ],
          "title": "remarks"
        }
      ],
      "links": [
        {
          "value": "https://rdap.apnic.net/autnum/9269",
          "rel": "self",
          "href": "https://rdap.apnic.net/entity/IRT-HKBN-HK",
          "type": "application/rdap+json"
        }
      ],
      "vcardArray": [
        "vcard",
        [
          [
            "version",
            {},
            "text",
            "4.0"
          ],
          [
            "fn",
            {},
            "text",
            "IRT-HKBN-HK"
          ],
          [
            "kind",
            {},
            "text",
            "group"
          ],
          [
            "adr",
            {
              "label": "15/F Trans Asia Centre\n18 Kin Hong Street, Kwai Chung\nN.T."
            },
            "text",
            [
              "",
              "",
              "",
              "",
              "",
              "",
              ""
            ]
          ],
          [
            "email",
            {},
            "text",
            "hostmaster@hkbn.com.hk"
          ],
          [
            "email",
            {
              "pref": "1"
            },
            "text",
            "abuse@hkbn.net"
          ]
        ]
      ],
      "objectClassName": "entity",
      "handle": "IRT-HKBN-HK"
    },
    {
      "roles": [
        "technical",
        "administrative"
      ],
      "events": [
        {
          "eventAction": "registration",
          "eventDate": "2012-05-23T03:52:23Z"
        },
        {
          "eventAction": "last changed",
          "eventDate": "2021-03-25T09:01:18Z"
        }
      ],
      "links": [
        {
          "value": "https://rdap.apnic.net/autnum/9269",
          "rel": "self",
          "href": "https://rdap.apnic.net/entity/HKBN-HK",
          "type": "application/rdap+json"
        }
      ],
      "vcardArray": [
        "vcard",
        [
          [
            "version",
            {},
            "text",
            "4.0"
          ],
          [
            "fn",
            {},
            "text",
            "HKBN Hostmaster"
          ],
          [
            "kind",
            {},
            "text",
            "individual"
          ],
          [
            "adr",
            {
              "label": "Trans Asia Centre\n18 Kin Hong Street, Kwai Chung\nN.T."
            },
            "text",
            [
              "",
              "",
              "",
              "",
              "",
              "",
              ""
            ]
          ],
          [
            "tel",
            {
              "type": "voice"
            },
            "text",
            "+852-3999-3888"
          ],
          [
            "tel",
            {
              "type": "fax"
            },
            "text",
            "+852-8167-7020"
          ],
          [
            "email",
            {},
            "text",
            "hostmaster@hkbn.com.hk"
          ],
          [
            "email",
            {
              "pref": "1"
            },
            "text",
            "abuse@hkbn.com.hk"
          ]
        ]
      ],
      "objectClassName": "entity",
      "handle": "HKBN-HK"
    }
  ],
  "port43": "whois.apnic.net"
}