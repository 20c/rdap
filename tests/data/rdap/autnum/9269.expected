{
  "name": "HKBN-AS-AP",
  "emails": [
    "abuse@hkbn.net",
    "hostmaster@hkbn.com.hk",
    "nocsn@hkbn.com.hk"
  ],
  "org_name": "Hong Kong Broadband Network Ltd, Hong Kong Broadband Network Ltd.",
  "org_address": "15/F Trans Asia Centre\n18 Kin Hong Street"
}